lib/parser.js

```diff
@@ -50,7 +50,8 @@ module.exports = {
     else parsed.type = 'ban'
 
     parsed.channel = channel
-    parsed.target_username = target_username.replace(/\r\n/g, '')
+    if (target_username)
```

Comment:
This is really my fault since I haven't set up an eslint config yet, but could you change it to use brackets please! 👍 

```javascript
if(target_username) {
  parsed.target_username = target_username.replace(/\r\n/g, '')
}
```